transports/msmq/index.md

```diff
@@ -77,36 +77,36 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 |----------------|---------------------|-----------------------------------|
 | Owning account | Send, Receive, Peek | Set by NServiceBus                |
 | Administrators | Full                | Set by NServiceBus                |
-| Anonymous      | Send                | Set by NServiceBus prior to 6.1.0 |
-| Everyone       | Send                | Set by NServiceBus prior to 6.1.0 |
+| Anonymous      | Send                | Set by NServiceBus in Versions 6.0.x and below |
+| Everyone       | Send                | Set by NServiceBus in Versions 6.0.x and below |
 
+NOTE: In Versions 6.1.0 and above, the installers will not automatically grant permissions to the `Anonymous` and `Everyone` group. It will respect the existing queue permissions that have been set up for the endpoint. The permissions granted to these user accounts is based on standard windows behavior and not via NServiceBus.
```

Comment:
I don't have a preference for either way. I did this to differentiate it in a consistent manner.